WORKSPACE

```diff
@@ -685,7 +685,7 @@ proto_library(
     visibility = ["//visibility:public"],
 )
 """,
-    sha256 = "e2dc7ad98f2bc1a78442a3e20eeef0381be008c18bf22f0dcb56283981977e01",
+    sha256 = "9cfede22ded095d600d9cc68a4aa83fe210e2628b2a81bf91f4de2d576a50984",
```

Comment:
@aherrmann-da @cocreature can we move this SHA to `bazel_tools/scalapb.bzl` so it is closer to where you are upticking the scalapb version and it is clearer it needs to move in tandem?